tox.ini

```diff
@@ -30,7 +30,7 @@ setenv =
     # "tox -e py37-coverage".
     coverage: _PYTEST_TOX_COVERAGE_RUN=coverage run -m
     coverage: _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
-    coverage: COVERAGE_FILE={toxinidir}/.coverage
```

Comment:
I think this was correct already 🤔 -- iirc `coverage` picks up the `COVERAGE_FILE` environment variable

it's unclear to me why there's custom code in github actions for this though 🤔

Reply:
Yes, the dev env was already setup correctly. This discrepancy, I guess, came out from the fact that we want to report coverage in a separate GHA step? Plus, we don't need upload coverage locally?